client/controllers/WebPublisherSettingsController.js

```diff
@@ -221,17 +402,51 @@ export function WebPublisherSettingsController($scope, publisher, modal, vocabul
                         {name: '<=', value: '<='},
                         {name: '>=', value: '>='}
                     ],
-                    category: [
-                        {name: '=', value: 'matches'},
+                    in: [
+                        {name: '=', value: 'in'},
+                    ],
+                    custom: [
+                        {name: '=', value: '=='},
+                        {name: '!=', value: '!='}
                     ]
                 }
             };
 
+            if($scope.newRule.type == 'organization') {
+                this.expressionBuilder.options = [
+                    {name: 'Language', value: 'package.getLanguage()', type: 'string'},
+                    {name: 'Category', value: 'package.getServices()', type: 'in'},
+                    {name: 'Author', value: 'package.getAuthorsNames()', type: 'in'},
+                    {name: 'Ingest Source', value: 'package.getSource()', type: 'string'},
+                    {name: 'Priority', value: 'package.getPriority()', type: 'number'},
+                    {name: 'Urgency', value: 'package.getUrgency()', type: 'number'},
+                ];
+            } else {
+                // article
+                customRuleFunctionName = 'article.getPackage().getExtra()';
+
+                this.expressionBuilder.options = [
+                    {name: 'Language', value: 'article.getLocale()', type: 'string'},
+                    {name: 'Category', value: 'article.getServices()', type: 'in'},
```

Comment:
here it should be `article.getPackage().getServices()` or `article.getMetadataByKey('service')`